tests/test_grid3d/test_grid.py

```diff
@@ -43,9 +43,8 @@
 
 
 @pytest.fixture()
-def load_gfile1():
-    """Fixture for loading EMEGFILE grid."""
-    return xtgeo.grid3d.Grid(EMEGFILE)
+def emerald_grid(testpath):
+    return xtgeo.grid3d.Grid(join(testpath, "3dgrids/eme/1/emerald_hetero_grid.roff"))
 
 
 def test_import_wrong(tmp_path):
```

Comment:
Consider renaming to `test_import_wrong_fformat` and removing docstring, especially the "etc." part.

Reply:
Done